pkg/acs/calacs/acsccd/doccd.c

```diff
@@ -81,22 +91,35 @@ int DoCCD (ACSInfo *acs_info) {
     void TimeStamp (char *, char *);
     void UCalVer (Hdr *);
     void UFilename (char *, Hdr *);
-    int FindOverscan (ACSInfo *, int, int, int *);
+    int FindOverscan (ACSInfo *, int, int, int *, int *);
     int GetCCDTab (ACSInfo *, int, int);
     int GetKeyBool (Hdr *, char *, int, Bool, Bool *);
 
     /*========================Start Code here =======================*/
 
+    PtrRegister ptrReg;
+    initPtrRegister(&ptrReg);
+
     /* set up an array of SingleGroups to hold all image extensions */
+    SingleGroup * x = NULL;   /* used for both input and output */
     x = (SingleGroup *) malloc(acs_info->nimsets * sizeof(SingleGroup));
```

Comment:
See comment above about instantiation, you shouldn't be allocating for a struct like this.